pkg/generate/app/app.go

```diff
@@ -47,6 +47,13 @@ func (s NameSuggestions) SuggestName() (string, bool) {
 	return "", false
 }
 
+// IsParameterizableValue returns true if the value contains standard replacement
+// syntax, to preserve the value for use inside of the generated output. Passing
+// parameters into output is only valid if the output is used inside of a template.
+func IsParameterizableValue(s string) bool {
+	return strings.Contains(s, "${") || strings.Contains(s, "$(")
```

Comment:
Why?

On Mon, Apr 18, 2016 at 11:04 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/generate/app/app.go
> <https://github.com/openshift/origin/pull/8535#discussion_r60167825>:
>
> > @@ -47,6 +47,13 @@ func (s NameSuggestions) SuggestName() (string, bool) {
> >  	return "", false
> >  }
> >
> > +// IsParameterizableValue returns true if the value contains standard replacement
> > +// syntax, to preserve the value for use inside of the generated output. Passing
> > +// parameters into output is only valid if the output is used inside of a template.
> > +func IsParameterizableValue(s string) bool {
> > +	return strings.Contains(s, "${") || strings.Contains(s, "$(")
>
> seems like a weak check.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8535/files/fbbcba5bad356141708270ae1ff6edadc069ca4f#r60167825>
>